### DISCUSSION TITLE
How do I get the Galaxy Brain badge?

### DISCUSSION THREAD

**Discussion by FWHassloch** [community/community#129373] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How do I get the Galaxy Brain badge?

**Comment by Flash2over** [comment, marked as answer]
After answering two discussions, You will get Galaxy Brain badge

**Reply by LiteBrite82** [reply]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
